src/read_csv.py

```diff
@@ -1,11 +1,15 @@
 import csv
-countylist = []
-newspaperlist = []
+import sys
+import os
+
+
 def readFile(file):

```

Comment:
Make this function name more descriptive. Think about what exactly is it reading in.